data/categorized/bowl/United States West Indies/Xavier Marshall  (United States  West Indies).csv

```diff
@@ -0,0 +1,3 @@
+,Match_id,Score,Overs,Maidens,Wicket,Economy
```

Comment:
Same problem, you can ignore USA as a country as of now, and simply change it to West Indies